fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -1847,6 +1848,11 @@ public Void visitDictQueryExpr(DictQueryExpr node, Scope context) {
         public Void visitParameterExpr(Parameter node, Scope context) {
             return null;
         }
+
+        @Override
+        public Void visitNamedArgument(NamedArgument node, Scope context) {
+            return null;
```

Comment:
Should we analyze the expr field in the NamedArgument node?

Reply:
no need, as it just holds parameters to function parameters, and then departs expressions and names in functionParam where to analyze expressions.